src/app/pages/home/services/tasks.service.ts

```diff
@@ -130,6 +59,39 @@ export class HomeController {
       return this.checkIfAllTasksComplete(tasks)
     })
   }
+
+  isLastTask(task) {
+    return this.getTasksOfToday().then((tasks: Task[]) => {
+      return this.checkIfLastTask(tasks, task)
+    })
+  }
+
+  checkIfAllTasksComplete(tasks: Task[]) {
+    if (tasks) {
+      for (let i = 0; i < tasks.length; i++) {
+        if (tasks[i].name !== 'ESM') {
+          if (tasks[i].completed === false) {
+            return false
+          }
+        }
+      }
+    }
+    return true
+  }
+
+  checkIfLastTask(tasks: Task[], task) {
+    if (tasks) {
+      for (let i = 0; i < tasks.length; i++) {
+        if (tasks[i].name !== 'ESM') {
+          if (tasks[i].completed === false && tasks[i].index !== task.index) {
+            return false
+          }
+        }
+      }
+    }
+    return true
+  }
```

Comment:
I think this can be merged with the isLastTask() function above seeing as this is not used elsewhere.